pyseir/cli.py

```diff
@@ -19,6 +19,7 @@
 from pyseir.deployment.webui_data_adaptor_v1 import WebUIDataAdaptorV1
 from libs.datasets import combined_datasets
 from pyseir.inference.whitelist_generator import WhitelistGenerator
+from pyseir.utils import get_summary_artifact_path, SummaryArtifact
```

Comment:
I would either import this as `import pyseir.utils` or `from pyseir import utils`

Reply:
Also, can you make these two separate lines? (soon to be a lint rule 🪂 )

Reply:
I went with `import pyseir.utils`, but that makes some very long function names. But I agree that pyseir.utils.X is needed to know where it comes from.